### PR TITLE
Update sample entry in Pacbio pos processing

### PR DESCRIPTION
**Pull request by diitaz93**
## Description
The Pacbio post-processing was missing updating the sample's reads and last sequenced date. This PR adds this step to the post-processing and updates the unit tests

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/store/crud/update.py [hunk 1]
```diff
@@ -77,6 +77,11 @@ def update_sample_reads_illumina(self, internal_id: str, sequencer_type: Sequenc
         sample.reads = total_reads_for_sample
         self.session.commit()
 
+    def update_sample_reads(self, internal_id: str, reads: int):
```
Comment (islean) [inline cg/store/crud/update.py:80]: Will this method work if a sample is spread over several runs? Would we not want to add the reads to any prior ones?

**Review by islean — APPROVED** [review]
Looks good!

File: cg/store/crud/update.py (hunk 1 above)
Comment (diitaz93) [reply cg/store/crud/update.py:80]: That is a good point. We will implement that when we have pooled samples

Comment (diitaz93) [reply cg/store/crud/update.py:80]: implementing +=

**Pull request merged**